v5/Http/Controllers/PostController.php

```diff
@@ -406,16 +406,23 @@ protected function savePostValues(Post $post, array $post_content, int $post_id)
                 continue;
             }
             foreach ($stage['fields'] as $field) {
-                $type = $field['type'];
-                $value = is_array($field['value']) && array_key_exists('value', $field['value'])
-                    ? $field['value']['value']
-                    : $field['value']; // Fall back to the field value if "value" key doesn't exist
-                if (!isset($value) || (is_array($value) && empty($value))) {
+                if (!isset($field['value'])) {
+                    continue;
+                }
+
+                // We only want to check if a field input value is set, not if it's empty
```

Comment:
After checking with you @webong , this makes sense 👍  

So, if we PUT a post and there’s an attribute in the body with `"value": { "value": null }` , that means the attribute will not be touched (essentially behaving  a bit like a PATCH request)

But if it’s `"value": { "value": {} }`, that should clear the attribute's value.

My only comment is then that I think I would have understood the comment more quickly if I was given examples as coming from the API .. instead of remembering what PHP considers set or empty.